### DISCUSSION TITLE
How to deploy local Quay server using the docker-compose.yml file

### DISCUSSION THREAD

**Discussion by LichtMaster** [quay/quay#1000] [Q&A]
For the past week I am failing at setting up a Quay instance using the provided docker-compose file in this repository https://github.com/quay/quay/blob/master/docker-compose.yaml.

When I try to access Quay from 8443(https) I receive connection refused.
Also when I try to access Quay from 8080 I receive "The Quay application could not be loaded, which typically indicates an external library could not be loaded (usually due to an ad blocker). Please check the JavaScript console for errors." **(I  don't use adblocker**

**Comment by HammerMeetNail** [comment]
Can you provide a little more information? How are you starting the containers and what OS are you using?

The [Getting Started](https://github.com/quay/quay/blob/master/docs/getting-started.md#starting-quay) doc suggests using `make local-dev-up`.
